2019-11-05 13:06:45,106 - experiment.py - INFO - =====================================================
2019-11-05 13:06:45,106 - experiment.py - INFO - Experiment ../experiments/cnn_reliability_0.json 5 started...
2019-11-05 13:06:45,106 - experiment.py - INFO - =====================================================
2019-11-05 13:06:45,107 - util.py - INFO - CALL::__main__.experiment_model(selection_problem = 'reliability', selection_source = 'mbfc', selection_count = 15000, selection_random_seed = 13, selection_reject_minimum = 500, selection_overwrite = False, embedding_type = 'w2v', embedding_shape = 'sequence', embedding_overwrite = False, test_source = 'ng', model_type = 'cnn', model_arch_num = 2, model_layer_sizes = [[16, 300, 300], 16, 1], model_maxlen = 500, model_batch_size = 128, model_learning_rate = 0.001, model_epochs = 100, model_num = 5, verbose = False, params = {'type': 'model', 'selection_problem': 'reliability', 'selection_source': 'mbfc', 'selection_count': 15000, 'selection_random_seed': 13, 'selection_reject_minimum': 500, 'selection_overwrite': False, 'embedding_type': 'w2v', 'embedding_shape': 'sequence', 'embedding_overwrite': False, 'verbose': False, 'test_source': 'ng', 'model_type': 'cnn', 'model_arch_num': 2, 'model_layer_sizes': [[16, 300, 300], 16, 1], 'model_maxlen': 500, 'model_batch_size': 128, 'model_learning_rate': 0.001, 'model_epochs': 100, 'model_num': 5})
2019-11-05 13:06:45,107 - util.py - INFO - CALL::datasets.get_selection_set(problem = 'reliability', source = 'mbfc', count = 15000, random_seed = 13, reject_minimum = 500, overwrite = False, verbose = False)
2019-11-05 13:06:45,115 - util.py - INFO - CALL::datasets.create_selection_set_sources(target_count = 15000, reject_minimum = 500, overwrite = False, verbose = False)
2019-11-05 13:06:45,115 - util.py - INFO - Cached version found.
2019-11-05 13:06:45,116 - datasets.py - INFO - Creating binary selection selection_reliability_mbfc_13...
2019-11-05 13:06:45,116 - util.py - INFO - Cached version found.
2019-11-05 13:06:45,116 - datasets.py - INFO - Loading ../data/cache/selection_reliability_mbfc_13...
Empty DataFrame
Columns: [date, source, name, content, reliable]
Index: []
2019-11-05 13:06:45,253 - util.py - INFO - CALL::datasets.get_embedding_set(df =              date  ... reliable
0      2018-10-19  ...        1
1      2018-11-02  ...        1
2      2018-07-26  ...        1
3      2018-09-06  ...        1
4      2018-07-29  ...        1
...           ...  ...      ...
29995  2018-11-16  ...        0
29996  2018-08-30  ...        0
29997  2018-10-22  ...        0
29998  2018-09-24  ...        0
29999  2018-08-07  ...        0

[30000 rows x 5 columns], embedding_type = 'w2v', output_name = 'selection_reliability_mbfc_13', shaping = 'sequence', overwrite = False)
2019-11-05 13:06:45,254 - datasets.py - INFO - Creating w2v embedding selection_reliability_mbfc_13...
2019-11-05 13:06:45,254 - util.py - INFO - Cached version found.
2019-11-05 13:06:59,686 - util.py - INFO - CALL::datasets.get_test_embedding_set(problem = 'reliability', source = 'mbfc', source_test = 'ng', count = 15000, reject_minimum = 500, random_seed = 13, embedding_type = 'w2v', shaping = 'sequence')
2019-11-05 13:06:59,686 - util.py - INFO - CALL::datasets.create_selection_set_sources(target_count = 15000, reject_minimum = 500, overwrite = False, verbose = False)
2019-11-05 13:06:59,686 - util.py - INFO - Cached version found.
2019-11-05 13:06:59,687 - util.py - INFO - CALL::datasets.get_selection_set(problem = 'reliability', source = 'ng', count = 15000, random_seed = 13, reject_minimum = 500, overwrite = False, verbose = False)
2019-11-05 13:06:59,696 - util.py - INFO - CALL::datasets.create_selection_set_sources(target_count = 15000, reject_minimum = 500, overwrite = False, verbose = False)
2019-11-05 13:06:59,696 - util.py - INFO - Cached version found.
2019-11-05 13:06:59,697 - datasets.py - INFO - Creating binary selection selection_reliability_ng_13...
2019-11-05 13:06:59,697 - util.py - INFO - Cached version found.
2019-11-05 13:06:59,697 - datasets.py - INFO - Loading ../data/cache/selection_reliability_ng_13...
Empty DataFrame
Columns: [date, source, name, content, reliable]
Index: []
2019-11-05 13:06:59,808 - util.py - INFO - CALL::datasets.get_embedding_set(df =              date  ... reliable
0      2018-10-19  ...        1
1      2018-11-02  ...        1
2      2018-07-26  ...        1
3      2018-09-06  ...        1
4      2018-07-29  ...        1
...           ...  ...      ...
29995  2018-10-15  ...        0
29996  2018-03-16  ...        0
29997  2018-07-30  ...        0
29998  2018-06-07  ...        0
29999  2018-03-25  ...        0

[30000 rows x 5 columns], embedding_type = 'w2v', output_name = 'selection_reliability_ng_13', shaping = 'sequence', overwrite = False)
2019-11-05 13:06:59,808 - datasets.py - INFO - Creating w2v embedding selection_reliability_ng_13...
2019-11-05 13:06:59,809 - util.py - INFO - Cached version found.
2019-11-05 13:07:34,386 - util.py - INFO - CALL::cnn.train_test(X = array([[[ 1.45507812e-01],
        [-1.89208984e-02],
        [ 9.61914062e-02],
        ...,
        [ 2.15820312e-01],
        [-1.33789062e-01],
        [-6.49414062e-02]],

       [[ 8.44726562e-02],
        [-3.52859497e-04],
        [ 5.32226562e-02],
        ...,
        [ 5.44433594e-02],
        [ 1.40625000e-01],
        [ 3.06640625e-01]],

       [[-1.62353516e-02],
        [ 9.42382812e-02],
        [ 9.17968750e-02],
        ...,
        [-8.30078125e-02],
        [ 6.54296875e-02],
        [-2.94189453e-02]],

       ...,

       [[-1.72851562e-01],
        [ 2.79296875e-01],
        [ 1.06933594e-01],
        ...,
        [ 0.00000000e+00],
        [ 0.00000000e+00],
        [ 0.00000000e+00]],

       [[ 1.39648438e-01],
        [ 9.61914062e-02],
        [-1.30004883e-02],
        ...,
        [ 0.00000000e+00],
        [ 0.00000000e+00],
        [ 0.00000000e+00]],

       [[-4.61425781e-02],
        [-8.78906250e-03],
        [-4.84375000e-01],
        ...,
        [ 0.00000000e+00],
        [ 0.00000000e+00],
        [ 0.00000000e+00]]], dtype=float32), y = 0        1
1        1
2        1
3        1
4        1
        ..
29995    0
29996    0
29997    0
29998    0
29999    0
Name: reliable, Length: 30000, dtype: int64, arch_num = 2, layer_sizes = [[16, 300, 300], 16, 1], maxlen = 500, batch_size = 128, learning_rate = 0.001, epochs = 100, X_test = array([[[-0.03979492],
        [ 0.13183594],
        [-0.09423828],
        ...,
        [ 0.        ],
        [ 0.        ],
        [ 0.        ]],

       [[ 0.3046875 ],
        [ 0.12402344],
        [-0.26953125],
        ...,
        [ 0.        ],
        [ 0.        ],
        [ 0.        ]],

       [[-0.20800781],
        [-0.2109375 ],
        [ 0.30273438],
        ...,
        [ 0.        ],
        [ 0.        ],
        [ 0.        ]],

       ...,

       [[ 0.02441406],
        [-0.05493164],
        [-0.0559082 ],
        ...,
        [ 0.        ],
        [ 0.        ],
        [ 0.        ]],

       [[ 0.18261719],
        [ 0.13769531],
        [ 0.12792969],
        ...,
        [ 0.        ],
        [ 0.        ],
        [ 0.        ]],

       [[ 0.01831055],
        [-0.20996094],
        [-0.14257812],
        ...,
        [ 0.        ],
        [ 0.        ],
        [ 0.        ]]], dtype=float32), y_test = 0       0
1       0
2       1
3       0
4       0
       ..
1711    1
1712    1
1713    0
1714    1
1715    0
Name: reliable, Length: 1716, dtype: int64, name = 'selection_reliability_mbfc_13_cnn_2_5_500_128_0.001')
Using TensorFlow backend.
WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

2019-11-05 13:07:34,387 - deprecation_wrapper.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

2019-11-05 13:07:34,406 - deprecation_wrapper.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

2019-11-05 13:07:34,408 - deprecation_wrapper.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

2019-11-05 13:07:34,466 - deprecation_wrapper.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/optimizers.py:790: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:3376: The name tf.log is deprecated. Please use tf.math.log instead.

2019-11-05 13:07:34,470 - deprecation_wrapper.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:3376: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
2019-11-05 13:07:34,473 - deprecation.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/tensorflow/python/ops/nn_impl.py:180: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:986: The name tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
conv1d_1 (Conv1D)            (None, 500, 16)           4816      
_________________________________________________________________
flatten_1 (Flatten)          (None, 8000)              0         
_________________________________________________________________
dense_1 (Dense)              (None, 16)                128016    
_________________________________________________________________
dense_2 (Dense)              (None, 1)                 17        
=================================================================
Total params: 132,849
Trainable params: 132,849
Non-trainable params: 0
_________________________________________________________________
2019-11-05 13:07:47,039 - deprecation_wrapper.py - WARNING - From /home/tntech.edu/namartinda42/.conda/envs/tensorflow-gpu/lib/python3.7/site-packages/keras/backend/tensorflow_backend.py:986: The name tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

2019-11-05 13:07:47.289447: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-11-05 13:07:47.297445: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2394580000 Hz
2019-11-05 13:07:47.297632: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55581b82b8f0 executing computations on platform Host. Devices:
2019-11-05 13:07:47.297647: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2019-11-05 13:07:47.301204: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Could not dlopen library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /cm/shared/apps/cuda91/toolkit/9.1.85/extras/CUPTI/lib64:/cm/local/apps/cuda/libs/current/lib64:/cm/shared/apps/cuda91/toolkit/9.1.85/targets/x86_64-linux/lib:/cm/local/apps/gcc/6.3.0/lib:/cm/local/apps/gcc/6.3.0/lib64:/cm/shared/apps/openmpi/gcc/64/1.10.3/lib64:/cm/shared/apps/slurm/17.02.10/lib64/slurm:/cm/shared/apps/slurm/17.02.10/lib64
2019-11-05 13:07:47.301227: E tensorflow/stream_executor/cuda/cuda_driver.cc:318] failed call to cuInit: UNKNOWN ERROR (303)
2019-11-05 13:07:47.301249: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (node026): /proc/driver/nvidia/version does not exist
2019-11-05 13:07:47.426000: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
Train on 24000 samples, validate on 6000 samples
Epoch 1/100
 - 42s - loss: 0.5815 - binary_accuracy: 0.6857 - val_loss: 0.5569 - val_binary_accuracy: 0.7138
Epoch 2/100
 - 42s - loss: 0.4652 - binary_accuracy: 0.7782 - val_loss: 0.5134 - val_binary_accuracy: 0.7430
Epoch 3/100
 - 43s - loss: 0.3973 - binary_accuracy: 0.8173 - val_loss: 0.5354 - val_binary_accuracy: 0.7418
Epoch 4/100
 - 44s - loss: 0.3385 - binary_accuracy: 0.8504 - val_loss: 0.5710 - val_binary_accuracy: 0.7333
Epoch 5/100
 - 43s - loss: 0.2778 - binary_accuracy: 0.8828 - val_loss: 0.6374 - val_binary_accuracy: 0.7233

 128/1716 [=>............................] - ETA: 0s
 384/1716 [=====>........................] - ETA: 0s
 512/1716 [=======>......................] - ETA: 0s
 768/1716 [============>.................] - ETA: 0s
1024/1716 [================>.............] - ETA: 0s
1280/1716 [=====================>........] - ETA: 0s
1536/1716 [=========================>....] - ETA: 0s
1716/1716 [==============================] - 1s 375us/step
2019-11-05 13:11:23,422 - cnn.py - INFO - Test loss / test accuracy: 1.301846 / 0.490676
